app/javascript/components/pxe-image-type-form/pxe-image-type-form.schema.js

```diff
@@ -0,0 +1,52 @@
+import { componentTypes, validatorTypes } from '@@ddf';
+import { parseCondition } from '@data-driven-forms/react-form-renderer';
+
+const createSchema = (fields, edit, loadSchema) => {
+  const idx = fields.findIndex((field) => field.name === 'volume_type');
+  const supports = edit ? 'supports_cloud_volume' : 'supports_cloud_volume_create';
+
+  return ({
+    fields: [
+      {
+        component: componentTypes.TEXT_FIELD,
+        name: 'name',
+        id: 'name',
+        label: __('Name'),
+        isRequired: true,
+        validate: [{ type: validatorTypes.REQUIRED }],
+      },
+      {
+        component: componentTypes.SELECT,
+        name: 'type',
```

Comment:
The field name can be deducted from the process I described [below](https://github.com/ManageIQ/manageiq-ui-classic/pull/7595/files#r563614959)